Script/char.dat

```diff
@@ -2881,29 +3304,95 @@ petrusswife
     DefaultWisdom = 8;
     DefaultCharisma = 85;
     HairColor = rgb16(200, 96, 0);
+    ClothColor = rgb16(150, 0, 0);
     HeadBitmapPos = 112, 32;
     PostFix = "number 4";
     FriendlyReplies =
     {
-      3,
+      7,
       "\"Uncle Petrus is so kind. After my father's house burned down because of the disargeement they had had, Petrus took me here and made me his wife!\"",
-      "\"I am boooored!\"",
+      "\"Uncle Petrus has a *massive* sword! Tee hee.\"",
+      "\"I'm boooored!\"",
+      "\"Have you heard No. 5? No, because she can't speak! Tee hee.\"",
+      "\"Don't mind No. 3! She looks grumpy, but she has so many stories!\"",
+      "\"No. 1 is so clever it's scary. Really!\"",
       "\"Uncle Petrus says I am special! He says it to all his wives, but I know he means it for me. Tee hee.\"";
```

Comment:
She's not young, just dumb. :D Maybe I can think of something for her to say to make it clear she's not a teen...